frontend/src/employee-frontend/components/unit/TabApplicationProcess.tsx

```diff
@@ -0,0 +1,54 @@
+{
+  /*
+SPDX-FileCopyrightText: 2017-2020 City of Espoo
+
+SPDX-License-Identifier: LGPL-2.1-or-later
+*/
+}
+
+import React, { useContext } from 'react'
+import { ContentArea } from 'lib-components/layout/Container'
+import { UnitContext } from '../../state/unit'
+import { SpinnerSegment } from 'lib-components/atoms/state/Spinner'
+import ErrorSegment from 'lib-components/atoms/state/ErrorSegment'
+import Title from 'lib-components/atoms/Title'
+import { useTranslation } from '../../state/i18n'
+import _ from 'lodash'
```

Comment:
Unused import? 🤔

Reply:
Wonder why this was not caught by the linter